code/modules/mob/living/simple_animal/hostile/terror_spiders/chem.dm

```diff
@@ -8,26 +9,44 @@
 	metabolization_rate = 0.1
 
 /datum/reagent/terror_black_toxin/on_mob_life(mob/living/M)
-	if(volume < 15)
-		// bitten once, die slowly. Easy to survive a single bite - just go to medbay.
-		// total damage: 2/tick, human health 150 until crit, = 75 ticks, = 150 seconds = 2.5 minutes to get to medbay.
-		M.adjustToxLoss(2) // same damage/tick as tabun cycle 0 to 60
-	else if(volume < 30)
-		// bitten twice, die more quickly, muscle cramps make movement difficult. Call medics immediately.
-		// total damage: 4, human health 150 until crit, = 37.5 ticks, = 75s = 1m15s until death
-		M.adjustToxLoss(4)
-		M.Confused(3)
+	if(volume < 30)
+		// bitten once, die very slowly. Easy to survive a single bite - just go to medbay.
+		// total damage: 1/tick, human health 150 until crit, = 150 ticks, = 300 seconds = 5 minutes to get to medbay.
```

Comment:
You do realize all of these comments will be 100% wrong if someone changes the tickrate or mob process rate, right?